<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1017</width>
    <height>663</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="centralwidget_2">
    <property name="geometry">
     <rect>
      <x>45</x>
      <y>20</y>
      <width>851</width>
      <height>482</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="_9">
     <item>
      <widget class="QGraphicsView" name="graphicsView_5">
       <property name="minimumSize">
        <size>
         <width>640</width>
         <height>480</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="_10">
       <item>
        <widget class="QListWidget" name="inventoryList_5">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>200</height>
          </size>
         </property>
         <property name="toolTip">
          <string>当前元件库存</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="openChestButton_5">
         <property name="text">
          <string>打开宝箱</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="combineButton_5">
         <property name="text">
          <string>组合元件</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="testGateButton_5">
         <property name="text">
          <string>测试通关</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="spacer_5">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>100</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QStatusBar" name="statusbar_2">
    <property name="geometry">
     <rect>
      <x>-170</x>
      <y>20</y>
      <width>3</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1017</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
